src/test/regress/citus_tests/common.py

```diff
@@ -526,10 +530,12 @@ def make_conninfo(self, **kwargs) -> str:
     def conn(self, *, autocommit=True, **kwargs):
         """Open a psycopg connection to this server"""
         self.set_default_connection_options(kwargs)
-        return psycopg.connect(
+        conn = psycopg.connect(
             autocommit=autocommit,
             **kwargs,
         )
+        conn.add_notice_handler(notice_handler)
```

Comment:
Is it possible to do the same for aconn?

Reply:
sadly no :(